vendor/github.com/intel/govmm/qemu/qmp.go

```diff
@@ -881,3 +881,13 @@ func (q *QMP) ExecHotplugMemory(ctx context.Context, qomtype, id, mempath string
 
 	return err
 }
+
+// ExecutePCIVSockAdd adds a vhost-vsock-pci bus
+func (q *QMP) ExecutePCIVSockAdd(ctx context.Context, id, guestCID string) error {
```

Comment:
Assuming we want to wait for the vsock device to appear in the agent, we should also control its pci bus address during hotplug.

Reply:
as a first step I think the current implementation is enough, I wouldn't like to add options/features that probably we won't need

Reply:
In that case, why not just drop `ExecutePCIVSockAdd`? You do not need hotplug at the first step, right? IMO, `ExecutePCIVSockAdd` here is not enough for hotplug anyway. And when we add vsock hotplug support as in [the following up steps we discussed](https://github.com/kata-containers/runtime/issues/532), we should add one that can control the pci bus address.